nextstrain/cli/runner/managed_conda.py

```diff
@@ -0,0 +1,337 @@
+"""
+Run commands with access to a fully-managed Conda environment.
+"""
+
+import os
+import platform
+import requests
+import shutil
+import subprocess
+import tarfile
+import traceback
+from pathlib import Path, PurePosixPath
+from typing import Iterable, Tuple
+from urllib.parse import urljoin
+from ..types import RunnerSetupStatus, RunnerTestResults, RunnerUpdateStatus
+from ..util import capture_output, exec_or_return, user_app_data_dir, warn
+
+
+USER_APP_DATA_DIR = user_app_data_dir()
+
+PREFIX     = USER_APP_DATA_DIR / "runtimes/managed-conda"
+PREFIX_BIN = PREFIX / "bin"
+
+MICROMAMBA_ROOT = USER_APP_DATA_DIR / "micromamba"
+MICROMAMBA      = MICROMAMBA_ROOT / "bin/micromamba"
+
+
+def register_arguments(parser) -> None:
+    """
+    No-op.  No arguments necessary.
+    """
+    pass
+
+
+def run(opts, argv, working_volume = None, extra_env = {}, cpus: int = None, memory: int = None) -> int:
+    if working_volume:
+        os.chdir(str(working_volume.src))
+
+    # XXX TODO: In the future we might want to set rlimits based on cpus and
+    # memory, at least on POSIX systems.
+    #   -trs, 21 May 2020 (copied from ./native.py on 30 Aug 2022)
+
+    extra_env.update({
+        "PATH": path_with_prefix(extra_env),
```

Comment:
I'm trying to piece together what's going on here - is this achieving the same end-goal as `conda activate <env-name>` in that it's setting the PATH to include where micromamba's installed things?

Reply:
It is similar to what `conda activate <name>` does but not identical. Importantly, modifying `PATH` is the piece that matters most for our usage.